src/adapters/audienceNetwork.js

```diff
@@ -131,33 +162,43 @@ const callBids = bidRequest => {
   const placementids = [];
   const adformats = [];
   const sizes = [];
+  const sdk = [];
+
   bidRequest.bids
     .filter(validateBidRequest)
     .forEach(bid => bid.sizes
       .map(flattenSize)
-      .filter(isValidSize)
+      .filter(size => isValidSize(size) || isVideo(bid.params.format))
       .slice(0, 1)
       .forEach(size => {
         adUnitCodes.push(bid.placementCode);
         placementids.push(bid.params.placementId);
         adformats.push(bid.params.format || size);
         sizes.push(size);
+        sdk.push(sdkVersion(bid.params.format));
       })
     );
 
   if (placementids.length) {
     // Build URL
     const testmode = isTestmode();
+    const pageurl = location.href;
+    const search = {
+      placementids,
+      adformats,
+      testmode,
+      pageurl,
+      sdk
+    };
+    const video = adformats.findIndex(isVideo);
```

Comment:
Looks like this has [browser compatibility issues](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex?v=example)